x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/interval_time_range_selector.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Dispatch, FC } from 'react';
+import { EuiSwitch } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface IntervalTimerangeSelectorProps {
+  setAddIntervalTimerange: Dispatch<React.SetStateAction<boolean>>;
+  addIntervalTimerange: boolean;
+  disabled: boolean;
+}
+
+/*
+ * React component for the form for adding a custom time range.
+ */
+export const IntervalTimerangeSelector: FC<IntervalTimerangeSelectorProps> = ({
+  setAddIntervalTimerange,
+  addIntervalTimerange,
+  disabled,
+}) => {
+  return (
+    <EuiSwitch
```

Comment:
Looks like you still need to get the 'Test' functionality working for the interval based URLs. Can you use the first result doc for testing?

Reply:
Fixed test url functionality in b1d2253b2e244beaa70fd36f16a44617455ce5d0 - should be working for custom time range and interval time range. 🙏